package.json

```diff
@@ -2,7 +2,7 @@
   "name": "meetup-web-platform",
   "version": "0.0.0",
```

Comment:
that actually gets set by Travis each time it pushes a release to npm, e.g. https://github.com/meetup/meetup-web-platform/commit/bfac5ab68d1839d428e4e364ec2df0c11b344a9a

it remains `0.0.0` in `master`